Exercises-4-OOP/src/ExercisesOnClasses/Rectangle.java

```diff
@@ -0,0 +1,48 @@
+package ExercisesOnClasses;
+//Ex: The Rectangle Class.
+
+public class Rectangle {
```

Comment:
The `Rectangle` class is not defined correctly.
You wrote:
```java
private float length = 5.6f;
private float width = 7.8f;
```

Meaning every instance of `Rectangle` will have the same `length` and `width`.
We should be able to create different instances:
Rectangle r1 = new Rectangle(1.2f, 3.4f);
Rectangle r2 = new Rectangle(6.5f, 4.0f);